app/controllers/api/v0/events_controller.rb

```diff
@@ -33,8 +33,11 @@ def convert_api_data_to_kafka_data(api_data:)
       # Adjust the client's occurred at time by a device sent at adjustment
       data[:occurred_at] = TimeUtil.infer_actual_occurred_at_from_client_timestamps(
         request_received_at: request.env[:received_at],
-        client_clock_occurred_at: api_data[:client_clock_occurred_at],
-        client_clock_sent_at: api_data[:client_clock_sent_at]).to_i
+        client_clock_occurred_at: data[:client_clock_occurred_at],
+        client_clock_sent_at: data[:client_clock_sent_at]
+      ).to_i
+
+      data.except!(:client_clock_sent_at, :client_clock_occurred_at)
```

Comment:
this should be tested...that we send a client_x data field but just get back a occurred_at field.

Reply:
added spec